Open5GS daemon v2.4.5-61-g7a9d060

05/10 16:39:05.070: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 16:39:05.070: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 16:39:05.076: [pfcp] WARNING: unknown key `advertise` (../lib/pfcp/context.c:244)
05/10 16:39:05.077: [sock] ERROR: socket bind(2) [172.22.0.2]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-socket.c:109)
05/10 16:39:05.077: [sock] ERROR: udp_server() [172.22.0.2]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-udp.c:67)
05/10 16:39:05.077: [app] ERROR: Failed to intialize UPF (../src/upf/app.c:28)
05/10 16:39:05.077: [app] FATAL: Open5GS initialization failed. Aborted (../src/main.c:219)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 16:40:46.520: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 16:40:46.520: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 16:40:46.577: [sock] ERROR: socket bind(2) [172.22.0.2]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-socket.c:109)
05/10 16:40:46.577: [sock] ERROR: udp_server() [172.22.0.2]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-udp.c:67)
05/10 16:40:46.577: [app] ERROR: Failed to intialize UPF (../src/upf/app.c:28)
05/10 16:40:46.577: [app] FATAL: Open5GS initialization failed. Aborted (../src/main.c:219)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 16:41:09.341: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 16:41:09.341: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 16:41:09.359: [sock] ERROR: socket bind(2) [172.22.0.2]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-socket.c:109)
05/10 16:41:09.359: [sock] ERROR: udp_server() [172.22.0.2]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-udp.c:67)
05/10 16:41:09.359: [app] ERROR: Failed to intialize UPF (../src/upf/app.c:28)
05/10 16:41:09.359: [app] FATAL: Open5GS initialization failed. Aborted (../src/main.c:219)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 16:43:04.912: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 16:43:04.912: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 16:43:04.921: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 16:43:04.926: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 16:43:04.926: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 17:27:00.189: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 17:27:00.189: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 17:27:00.205: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 17:27:00.206: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 17:27:00.206: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 18:10:03.197: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 18:10:03.197: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 18:10:03.213: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 18:10:03.213: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 18:10:03.213: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 18:17:24.513: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 18:17:24.513: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 18:17:24.522: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 18:17:24.522: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 18:17:24.522: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 18:19:57.477: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 18:19:57.477: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 18:19:57.494: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 18:19:57.494: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 18:19:57.497: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 18:21:47.567: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 18:21:47.567: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 18:21:47.575: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 18:21:47.575: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 18:21:47.575: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 18:24:34.637: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 18:24:34.637: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 18:24:34.661: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 18:24:34.661: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 18:24:34.661: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 18:27:58.531: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 18:27:58.532: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 18:27:58.539: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 18:27:58.539: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 18:27:58.539: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 18:29:34.181: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 18:29:34.181: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 18:29:34.207: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 18:29:34.207: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 18:29:34.208: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/10 18:32:35.730: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/10 18:32:35.730: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/10 18:32:35.739: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/10 18:32:35.739: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/10 18:32:35.739: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/11 11:35:41.873: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/11 11:35:41.873: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/11 11:35:41.882: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/11 11:35:41.882: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/11 11:35:41.882: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/11 14:17:35.352: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/11 14:17:35.352: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/11 14:17:35.361: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/11 14:17:35.361: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/11 14:17:35.361: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/11 15:06:34.015: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/11 15:06:34.015: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/11 15:06:34.021: [pfcp] WARNING: unknown key `advertise` (../lib/pfcp/context.c:244)
05/11 15:06:34.038: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/11 15:06:34.038: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/11 15:06:34.041: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/11 15:06:34.179: [pfcp] INFO: ogs_pfcp_connect() [172.22.0.1]:41300 (../lib/pfcp/path.c:61)
05/11 15:06:34.179: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/11 15:12:44.879: [pfcp] WARNING: [33] LOCAL  No Reponse. Give up! for step 1 type 1 peer [172.22.0.1]:41300 (../lib/pfcp/xact.c:611)
05/11 15:12:44.879: [upf] WARNING: No Heartbeat from SMF [172.22.0.1]:41300 (../src/upf/pfcp-sm.c:256)
05/11 15:12:44.879: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
Open5GS daemon v2.4.5-61-g7a9d060

05/11 15:13:27.784: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/11 15:13:27.784: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/11 15:13:27.792: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/11 15:13:27.792: [pfcp] INFO: ogs_pfcp_connect() [10.0.20.49]:8805 (../lib/pfcp/path.c:61)
05/11 15:13:27.794: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/11 15:13:27.794: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/11 15:13:30.566: [pfcp] INFO: ogs_pfcp_connect() [172.22.0.1]:39199 (../lib/pfcp/path.c:61)
05/11 15:13:30.567: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/11 15:13:35.300: [pfcp] WARNING: [1] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:13:38.793: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:13:46.309: [pfcp] WARNING: [2] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:13:49.794: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
Open5GS daemon v2.4.5-61-g7a9d060

05/11 15:13:55.011: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/11 15:13:55.011: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/11 15:13:55.018: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/11 15:13:55.018: [pfcp] INFO: ogs_pfcp_connect() [10.0.20.49]:8805 (../lib/pfcp/path.c:61)
05/11 15:13:55.019: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/11 15:13:55.019: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/11 15:14:02.525: [pfcp] WARNING: [1] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:14:06.023: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:14:13.532: [pfcp] WARNING: [2] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:14:17.028: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:14:24.537: [pfcp] WARNING: [3] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:14:28.033: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:14:35.541: [pfcp] WARNING: [4] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:14:39.034: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:14:46.543: [pfcp] WARNING: [5] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:14:50.038: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:14:57.544: [pfcp] WARNING: [6] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:15:01.042: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:15:08.550: [pfcp] WARNING: [7] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:15:12.046: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:15:19.560: [pfcp] WARNING: [8] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:15:23.050: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:15:30.557: [pfcp] WARNING: [9] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:15:34.057: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
05/11 15:15:41.566: [pfcp] WARNING: [10] LOCAL  No Reponse. Give up! for step 1 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/11 15:15:45.062: [upf] WARNING: Retry to association with peer [10.0.20.49]:8805 failed (../src/upf/pfcp-sm.c:106)
Open5GS daemon v2.4.5-61-g7a9d060

05/11 15:19:04.648: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/11 15:19:04.648: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/11 15:19:04.655: [pfcp] INFO: pfcp_server() [172.22.0.8]:8805 (../lib/pfcp/path.c:30)
05/11 15:19:04.655: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/11 15:19:04.665: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
Open5GS daemon v2.4.5-61-g7a9d060

05/11 16:13:03.947: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/11 16:13:03.947: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/11 16:13:03.957: [pfcp] WARNING: unknown key `advertise` (../lib/pfcp/context.c:244)
05/11 16:13:03.958: [pfcp] INFO: pfcp_server() [172.22.0.8]:8806 (../lib/pfcp/path.c:30)
05/11 16:13:03.958: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/11 16:13:03.958: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/11 16:13:09.099: [pfcp] INFO: ogs_pfcp_connect() [172.22.0.1]:34238 (../lib/pfcp/path.c:61)
05/11 16:13:09.099: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/11 16:13:57.257: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:164)
05/11 16:13:57.257: [gtp] INFO: gtp_connect() [10.0.23.211]:2152 (../lib/gtp/path.c:60)
05/11 16:13:57.257: [gtp] INFO: gtp_connect() [172.22.0.7]:2152 (../lib/gtp/path.c:60)
05/11 16:13:57.257: [upf] INFO: UE F-SEID[CP:0x1 UP:0x1] APN[internet] PDN-Type[1] IPv4[10.47.0.2] IPv6[] (../src/upf/context.c:367)
05/16 17:43:19.126: [pfcp] WARNING: [39749] LOCAL  No Reponse. Give up! for step 1 type 1 peer [172.22.0.1]:34238 (../lib/pfcp/xact.c:611)
05/16 17:43:19.128: [upf] WARNING: No Heartbeat from SMF [172.22.0.1]:34238 (../src/upf/pfcp-sm.c:256)
05/16 17:43:19.128: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
Open5GS daemon v2.4.5-61-g7a9d060

05/16 19:48:00.227: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/16 19:48:00.227: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/16 19:48:00.235: [pfcp] WARNING: unknown key `advertise` (../lib/pfcp/context.c:244)
05/16 19:48:00.236: [pfcp] INFO: pfcp_server() [172.22.0.8]:8806 (../lib/pfcp/path.c:30)
05/16 19:48:00.236: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/16 19:48:00.288: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/16 21:15:26.304: [pfcp] INFO: ogs_pfcp_connect() [10.0.20.49]:8805 (../lib/pfcp/path.c:61)
05/16 21:15:26.305: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/17 08:59:06.527: [pfcp] WARNING: [3836] LOCAL  No Reponse. Give up! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/17 08:59:06.527: [upf] WARNING: No Heartbeat from SMF [10.0.20.49]:8805 (../src/upf/pfcp-sm.c:256)
05/17 08:59:06.527: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/17 08:59:11.439: [upf] WARNING: cannot handle PFCP message type[1] (../src/upf/pfcp-sm.c:139)
05/17 08:59:11.439: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 08:59:11.439: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 08:59:11.439: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 08:59:11.439: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 08:59:11.439: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 08:59:11.439: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 08:59:20.248: [upf] WARNING: cannot handle PFCP message type[1] (../src/upf/pfcp-sm.c:139)
05/17 08:59:22.750: [pfcp] WARNING: [7673] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 08:59:25.253: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/17 09:00:27.785: [pfcp] WARNING: [3841] LOCAL  No Reponse. Give up! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/17 09:00:27.785: [upf] WARNING: No Heartbeat from SMF [10.0.20.49]:8805 (../src/upf/pfcp-sm.c:256)
05/17 09:00:27.785: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/17 09:00:39.804: [upf] WARNING: cannot handle PFCP message type[1] (../src/upf/pfcp-sm.c:139)
05/17 09:00:39.807: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 09:00:39.807: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 09:00:39.807: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 09:00:39.807: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 09:00:39.807: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 09:00:39.807: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 09:00:43.451: [pfcp] WARNING: [7685] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 09:00:45.952: [pfcp] WARNING: [7685] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 09:00:48.455: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/17 09:16:42.214: [pfcp] WARNING: [3927] LOCAL  No Reponse. Give up! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/17 09:16:42.214: [upf] WARNING: No Heartbeat from SMF [10.0.20.49]:8805 (../src/upf/pfcp-sm.c:256)
05/17 09:16:42.214: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/17 09:16:56.966: [upf] WARNING: cannot handle PFCP message type[1] (../src/upf/pfcp-sm.c:139)
05/17 09:16:56.967: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 09:16:56.968: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 09:16:56.968: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 09:16:56.968: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 09:16:56.968: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 09:16:56.968: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 09:16:59.466: [pfcp] WARNING: [7858] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 09:17:01.969: [pfcp] WARNING: [7858] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 09:17:04.471: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/17 14:04:29.877: [pfcp] WARNING: [10994] REMOTE Request Duplicated. Retransmit! for step 2 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:04:32.378: [pfcp] WARNING: [10994] REMOTE Request Duplicated. Retransmit! for step 2 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:04:34.878: [pfcp] WARNING: [5494] LOCAL  No Reponse. Give up! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/17 14:04:34.879: [upf] WARNING: PFCP[REQ] has already been associated (../src/upf/pfcp-sm.c:202)
05/17 14:04:34.879: [upf] WARNING: No Heartbeat from SMF [10.0.20.49]:8805 (../src/upf/pfcp-sm.c:256)
05/17 14:04:34.879: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/17 14:04:46.754: [upf] WARNING: cannot handle PFCP message type[1] (../src/upf/pfcp-sm.c:139)
05/17 14:04:49.256: [pfcp] WARNING: [10998] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 14:04:51.758: [pfcp] WARNING: [10998] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 14:04:54.260: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/17 14:46:24.649: [pfcp] ERROR: invalid step[0] type[2] (../lib/pfcp/xact.c:432)
05/17 14:46:24.649: [pfcp] ERROR: ogs_pfcp_xact_update_rx() failed (../lib/pfcp/xact.c:706)
05/17 14:46:38.153: [pfcp] WARNING: [11454] REMOTE Request Duplicated. Retransmit! for step 2 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:46:40.654: [pfcp] WARNING: [11454] REMOTE Request Duplicated. Retransmit! for step 2 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:46:43.155: [pfcp] WARNING: [5721] LOCAL  No Reponse. Give up! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/17 14:46:43.155: [upf] WARNING: PFCP[REQ] has already been associated (../src/upf/pfcp-sm.c:202)
05/17 14:46:43.155: [upf] WARNING: No Heartbeat from SMF [10.0.20.49]:8805 (../src/upf/pfcp-sm.c:256)
05/17 14:46:43.155: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/17 14:46:45.655: [pfcp] WARNING: [11455] REMOTE Request Duplicated. Retransmit! for step 2 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:46:48.156: [pfcp] WARNING: [11455] REMOTE Request Duplicated. Retransmit! for step 2 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:46:54.156: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/17 14:46:56.657: [pfcp] WARNING: [11457] REMOTE Request Duplicated. Retransmit! for step 2 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:46:59.158: [pfcp] WARNING: [11457] REMOTE Request Duplicated. Retransmit! for step 2 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:47:05.158: [upf] WARNING: PFCP[REQ] has already been associated (../src/upf/pfcp-sm.c:202)
05/17 14:47:07.658: [pfcp] WARNING: [11459] REMOTE Request Duplicated. Retransmit! for step 2 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:47:10.159: [pfcp] WARNING: [11459] REMOTE Request Duplicated. Retransmit! for step 2 type 5 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:390)
05/17 14:47:12.662: [pfcp] WARNING: [5722] LOCAL  No Reponse. Give up! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:611)
05/17 14:47:12.662: [upf] WARNING: No Heartbeat from SMF [10.0.20.49]:8805 (../src/upf/pfcp-sm.c:256)
05/17 14:47:12.662: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/17 14:47:21.429: [upf] WARNING: cannot handle PFCP message type[1] (../src/upf/pfcp-sm.c:139)
05/17 14:47:23.932: [pfcp] WARNING: [11462] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 14:47:26.435: [pfcp] WARNING: [11462] REMOTE Request Duplicated. Discard! for step 1 type 1 peer [10.0.20.49]:8805 (../lib/pfcp/xact.c:402)
05/17 14:47:28.937: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
Open5GS daemon v2.4.5-61-g7a9d060

05/17 14:52:24.132: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/17 14:52:24.132: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/17 14:52:24.139: [pfcp] WARNING: unknown key `advertise` (../lib/pfcp/context.c:244)
05/17 14:52:24.139: [pfcp] INFO: pfcp_server() [172.22.0.8]:8806 (../lib/pfcp/path.c:30)
05/17 14:52:24.139: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/17 14:52:24.145: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/17 14:52:36.027: [pfcp] INFO: ogs_pfcp_connect() [10.0.20.49]:8805 (../lib/pfcp/path.c:61)
05/17 14:52:36.027: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
Open5GS daemon v2.4.5-61-g7a9d060

05/17 15:29:33.882: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/17 15:29:33.882: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/17 15:29:33.891: [pfcp] INFO: pfcp_server() [172.22.0.8]:8806 (../lib/pfcp/path.c:30)
05/17 15:29:33.891: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/17 15:29:33.892: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/17 15:29:35.278: [pfcp] INFO: ogs_pfcp_connect() [172.22.0.1]:56217 (../lib/pfcp/path.c:61)
05/17 15:29:35.278: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/17 17:17:21.921: [pfcp] WARNING: [587] LOCAL  No Reponse. Give up! for step 1 type 1 peer [172.22.0.1]:56217 (../lib/pfcp/xact.c:611)
05/17 17:17:21.928: [upf] WARNING: No Heartbeat from SMF [172.22.0.1]:56217 (../src/upf/pfcp-sm.c:256)
05/17 17:17:21.928: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/17 17:18:18.229: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
Open5GS daemon v2.4.5-61-g7a9d060

05/17 17:21:07.672: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/17 17:21:07.672: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/17 17:21:07.724: [pfcp] INFO: pfcp_server() [172.22.0.8]:8806 (../lib/pfcp/path.c:30)
05/17 17:21:07.724: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/17 17:21:07.724: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/17 17:21:08.299: [pfcp] INFO: ogs_pfcp_connect() [172.22.0.1]:44008 (../lib/pfcp/path.c:61)
05/17 17:21:08.299: [upf] WARNING: cannot handle PFCP message type[1] (../src/upf/pfcp-sm.c:139)
05/17 17:21:10.801: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/17 18:13:39.462: [upf] WARNING: PFCP[REQ] has already been associated (../src/upf/pfcp-sm.c:202)
05/18 14:57:07.360: [pfcp] WARNING: [7065] LOCAL  No Reponse. Give up! for step 1 type 1 peer [172.22.0.1]:44008 (../lib/pfcp/xact.c:611)
05/18 14:57:07.364: [upf] WARNING: No Heartbeat from SMF [172.22.0.1]:44008 (../src/upf/pfcp-sm.c:256)
05/18 14:57:07.364: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/18 14:57:13.798: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
05/18 15:01:45.693: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:164)
05/18 15:01:45.699: [gtp] INFO: gtp_connect() [10.0.23.211]:2152 (../lib/gtp/path.c:60)
05/18 15:01:45.699: [gtp] INFO: gtp_connect() [172.22.0.7]:2152 (../lib/gtp/path.c:60)
05/18 15:01:45.699: [upf] INFO: UE F-SEID[CP:0x1 UP:0x1] APN[internet] PDN-Type[1] IPv4[10.45.0.2] IPv6[] (../src/upf/context.c:367)
05/18 15:25:35.834: [pfcp] WARNING: [7219] LOCAL  No Reponse. Give up! for step 1 type 1 peer [172.22.0.1]:44008 (../lib/pfcp/xact.c:611)
05/18 15:25:35.834: [upf] WARNING: No Heartbeat from SMF [172.22.0.1]:44008 (../src/upf/pfcp-sm.c:256)
05/18 15:25:35.834: [upf] INFO: PFCP de-associated (../src/upf/pfcp-sm.c:178)
05/18 15:26:18.633: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
Open5GS daemon v2.4.5-61-g7a9d060

05/18 15:45:42.881: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
05/18 15:45:42.881: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf.log' (../lib/app/ogs-init.c:132)
05/18 15:45:42.897: [pfcp] INFO: pfcp_server() [172.22.0.8]:8806 (../lib/pfcp/path.c:30)
05/18 15:45:42.897: [gtp] INFO: gtp_server() [172.22.0.8]:2152 (../lib/gtp/path.c:30)
05/18 15:45:42.898: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
05/18 15:45:44.038: [pfcp] INFO: ogs_pfcp_connect() [172.22.0.1]:36810 (../lib/pfcp/path.c:61)
05/18 15:45:44.039: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
